libs/util/nestjs/kafka/src/Kafka.module.ts

```diff
@@ -26,8 +25,7 @@ import { KafkaPubSubService } from "./kafka.pubsub.service";
       useClass: KafkaMessageJsonSerializer,
     },
     KafkaProducerService,
-    KafkaPubSubService,
   ],
-  exports: [KAFKA_SERIALIZER, KafkaProducerService, KafkaPubSubService],
+  exports: [KAFKA_SERIALIZER, KafkaProducerService],
```

Comment:
Removed the service from the kafkaModule otherwise all services using this will have a kafka consumer reading all the partition of the shared topic even if they never need it.